Lib/venv/__init__.py

```diff
@@ -178,6 +178,23 @@ def create_configuration(self, context):
             f.write('version = %d.%d.%d\n' % sys.version_info[:3])
             if self.prompt is not None:
                 f.write(f'prompt = {self.prompt!r}\n')
+            f.write('executable = %s\n' % os.path.realpath(sys.executable))
+            args = []
+            if self.symlinks:
+                args.append('--symlinks')
```

Comment:
I'd only append if the attribute value is different from the default (which is platform-dependent) - you wouldn't need to explicitly set a default value setting with a command-line argument, only to set a non-default value.